src/main/java/carpet/CarpetSettings.java

```diff
@@ -57,29 +58,25 @@
     public static int updateSuppressionBlockSetting = -1;
 
     private static class LanguageValidator extends Validator<String> {
-        @Override public String validate(CommandSourceStack source, ParsedRule<String> currentRule, String newValue, String string) {
-            if (currentRule.get().equals(newValue) || source == null)
```

Comment:
was that necessary for startup? Why is that removed now?

Reply:
This is because I've tried to shape this a bit for fallen's translation rework system, and started moving a few things to translations (to not have half of it deprecated right after release).

Previously carpet would just use the fallbacks from code in default language, now we always use the language system (the legacy, deprecated fallbacks are injected into the language system when registering), so we have to init it even with the default value.